pkg/config/config.go

```diff
@@ -283,15 +290,25 @@ func RunningInInstaller() bool {
 	return os.Getenv(model.OktetoInInstaller) == "true"
 }
 
-func IsInsecureSkipVerify() bool {
-	envVarValue := os.Getenv("OKTETO_INSECURE_SKIP_TLS_VERIFY")
-	value, err := strconv.ParseBool(envVarValue)
+func IsInsecureSkipVerify(getValue func() string) bool {
+	value := getValue()
+
+	if value == "" {
+		return false
+	}
+
+	insecureSkipVerify, err := strconv.ParseBool(value)
 
 	if err != nil {
-		oktetoLog.Debugf("insecure mode disabled, error when reading insecure mode: OKTETO_INSECURE_SKIP_TLS_VERIFY=%s, %v", envVarValue, err)
+		oktetoLog.Debugf("insecure mode disabled, error when reading insecure mode: %q, %v", value, err)
 		return false
 	}
 
-	oktetoLog.Debugf("insecure mode: %v", value)
-	return value
+	if insecureSkipVerify {
+		insecureSkipVerifyOnce.Do(func() { oktetoLog.Warning("Insecure mode enabled") })
+		return true
+	}
+
+	oktetoLog.Debug("insecure mode: %v", insecureSkipVerify)
```

Comment:
I think we should move this comment above the `if` before so we always get the log